configs/openmrs_config/ampathforms/adult_NCD_Consultation.json

```diff
@@ -155,7 +155,19 @@
 							},
 							"type": "obs",
 							"validators": []
-						}
+						},
+						{
+                            "id": "HBA1c",
+                            "questionOptions": {
+                                "rendering": "number",
+                                "concept": "a8af7520-1350-11df-a1f1-0026b9348838",
+                                "max": "20",
```

Comment:
Could you throw a little light why the maximum value is set to `20`?

Reply:
@reagan kindly advise on this